homeassistant/components/home_connect/coordinator.py

```diff
@@ -46,6 +47,7 @@ class HomeConnectApplianceData:
     info: HomeAppliance
     settings: dict[SettingKey, GetSetting]
     status: dict[StatusKey, Status]
+    events: dict[EventKey, Event] = field(default_factory=dict)
```

Comment:
I'd sort the attributes :abcd:.

Reply:
You'd set `events` over `info`?
I feel like `info` is the frontline of the class so that it should be first (but `events` should be moved above `settings` for sure)

Reply:
Just sort it :abcd:. Easier to find things then.